config/deploy.rb

```diff
@@ -1,13 +1,19 @@
 set :application, 'was-registrar'
 set :repo_url, 'https://github.com/sul-dlss/was-registrar.git'
+set :deploy_host, "was-registrar-#{fetch(:stage)}.stanford.edu"
+set :user, 'was'
 
 set :stages, %W(stage development production)
```

Comment:
shouldn't this be dev stage prod? i'm not sure what `:stages` is used for though...